turbinia/lib/docker_manager.py

```diff
@@ -42,6 +43,39 @@ def IsBlockDevice(path):
   return stat.S_ISBLK(mode)
 
 
+def GetDockerPath(mount_path):
+  """Retrieves the Docker installation path.
+
+  Args:
+    mount_path(str): The mount path of the Evidence object.
+
+  Returns:
+    docker_path(str): The Docker installation path.
+  """
+  docker_path = None
+  etc_path = os.path.join(mount_path, 'etc/docker/daemon.json')
+  if os.path.exists(mount_path):
+    try:
+      with open(etc_path) as etc_handle:
+        json_obj = json.loads(etc_handle.read())['data-root']
+        # Remove starting / so paths can join
+        if json_obj.startswith('/'):
+          json_obj = json_obj[1:]
+        docker_path = os.path.join(mount_path, json_obj)
+    except Exception as exception:
```

Comment:
This is a pretty broad exception to be catching, and this is usually only done in fairly rare circumstances.  Is there a more specific exception we can catch here?  Maybe IndexError?

Reply:
Updated to catch `FileNotFoundError` and `KeyError`, which should be the only two exceptions raised in this try/except.

Reply:
Removed `FileNotFoundError` as I'm checking to see if the etc config file path exists instead so that exception should never be raised.